### PR TITLE
Update to terraform scripts.

### PR DESCRIPTION
**Pull request by xpouyat**
Terraform script update thanks to @se02035

### CODE REVIEW

File: infrastructure/terraform/.terraform.lock.hcl [hunk 1]
```diff
@@ -0,0 +1,115 @@
+# This file is maintained automatically by "terraform init".
```
Comment (flecoqui) [inline infrastructure/terraform/.terraform.lock.hcl:1]: Could you remove this file from the PR?

Comment (flecoqui) [inline infrastructure/terraform/.terraform.lock.hcl:1]: Could you remove this file from the PR?

**Review by flecoqui — APPROVED** [review]
Approved provided you remove the lock file.

File: infrastructure/terraform/.terraform.lock.hcl (hunk 1 above)
Comment (xpouyat) [reply infrastructure/terraform/.terraform.lock.hcl:1]: @se02035 Should I remove it ? I saw you put it

Comment (se02035) [reply infrastructure/terraform/.terraform.lock.hcl:1]: It's best practice to have the lock file in the version control system. To better control terraforms dependency management.

Pls see this https://www.terraform.io/language/files/dependency-lock

@flecoqui @xpouyat

Comment (xpouyat) [reply infrastructure/terraform/.terraform.lock.hcl:1]: @flecoqui Ok to keep the file ?

**Pull request merged**